_includes/parse-server/deploying-aws-ubuntu.md

```diff
@@ -0,0 +1,192 @@
+## Deploying on AWS EC2 Ubuntu using PostgreSQL
+
+Here are the steps:
+
+1. Log into your AWS account or create a new one [AWS Account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/)
+2. Go to AWS EC2 Dashboard
+3. Launch Instances
+4. On **Application and OS Images (Amazon Machine Image)** select `Ubuntu`
+5. On **Instance Type** select `t2.micro` (is ok for testing and small projects which is Free tier eligible)
+6. Create or select an existing `key pair`. (If you create a new one click `Download Key Pair`)
+7. Click `Launch Instance`
+8. On **EC2 Dashboard** select the new created instance and click `Security`
+9. On **Security Groups**, click on the security group _i.e.:sg-0fg55kj5588d63_
```

Comment:
I don't think its necessary to add a security group example ID, maybe remove "_i.e.:sg-0fg55kj5588d63_"?